### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by qknight**
@zimbatm my proposal but please note my knowledge of direnv is limited to the flake/shell example I've tested on WSL2 in nixos-wsl. IMHO this new README.md outlines the capabilities of direnv much better.

Is there interactive shell support implemented/planned? I didn't have time to study the code yet, but I would love to try it with vscodium https://lastlog.de/blog/vscodium_and_nix_develop.html because it might just work.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,47 +6,30 @@ direnv -- unclutter your .profile
 [![latest packaged version(s)](https://repology.org/badge/latest-versions/direnv.svg)](https://repology.org/project/direnv/versions)
 [![Support room on Matrix](https://img.shields.io/matrix/direnv:numtide.com.svg?label=%23direnv%3Anumtide.com&logo=matrix&server_fqdn=matrix.numtide.com)](https://matrix.to/#/#direnv:numtide.com)
 
-`direnv` is an extension for your shell. It augments existing shells with a
-new feature that can load and unload environment variables depending on the
-current directory.
+`direnv` is an extension for your unix shell. It augments existing shells with a
```
Comment (zimbatm) [inline README.md:9]: Maybe "terminal shell"? Direnv also works with PowerShell.

File: README.md [hunk 2]
```diff
@@ -6,47 +6,30 @@ direnv -- unclutter your .profile
...
+new feature that can **load and unload environment variables** depending on the
+directory you are in.
 
 ## Use cases
 
-* Load [12factor apps](https://12factor.net/) environment variables
+Primary use-cases **hacking on nix based software**:
```
Comment (zimbatm) [inline README.md:15]: I don't know that this is the primary use case.

The principle of direnv is that it's an agnostic interface.
Many people use direnv with asdf, homebrew, or just standalone.

File: README.md [hunk 3]
```diff
@@ -6,47 +6,30 @@ direnv -- unclutter your .profile
...
+* **bash, zsh, fish, tcsh, csh, elvish, rc, powershell, murex, nushell and xonsh** shells.
 
-Because direnv is compiled into a single static executable, it is fast enough
-to be unnoticeable on each prompt. It is also language-agnostic and can be
-used to build solutions similar to rbenv, pyenv and phpenv.
-
-## Getting Started
-
-### Prerequisites
+Note: **direnv is limited to interactive shells** so 'ssh nixos@nixos "cd myproject; export" will not show added variables but 'ssh nixos@nixos' and then manually typing 'export' will show them.
```
Comment (zimbatm) [inline README.md:30]: But you can use `ssh nixos@nixos "cd myproject; direnv exec . export"`

**Review by zimbatm — COMMENTED** [review]
Thanks!

It goes in the right direction, but also is a bit too focused on Nix.

Do you want to give it another go?

File: README.md (hunk 1 above)
Comment (qknight) [reply README.md:9]: I think the original was alright. Just thought most users were on Unix.

Comment (qknight) [reply README.md:9]: I tried to get it working in PowerShell (or cmd.exe) but I couldn't get it working.

File: README.md (hunk 2 above)
Comment (qknight) [reply README.md:15]: I trust you here, as I said, I don't have a clue about the direnv users' environments.

File: README.md (hunk 3 above)
Comment (qknight) [reply README.md:30]: That is good to know but I'd consider this a workaround or hack rather than 'supported'.

**Comment by qknight** [comment]
I'm not sure how to make a useful contribution here. 
At least I'd love to have the `use flake` and `use nix` documented in the main README.md

**Pull request closed without merging**